docs/pages/access-controls/sso/google-workspace.mdx

```diff
@@ -170,78 +149,197 @@ correct Google Workspace plan.
-    fetch both direct and indirect groups or just direct groups, respectively.
+Add **one** of the following scopes (more info [here](#how-google-workspace-apis-differ-by-service-level)):
+- Direct and Indirect Groups (*Transitive Group Membership Support*)
+  - `https://www.googleapis.com/auth/cloud-identity.groups.readonly`
+- Direct Groups Only
+  - `https://www.googleapis.com/auth/admin.directory.group.readonly`
 
-## Step 3/4. Create an OIDC connector
+## Step 2/4. Create an OIDC connector
 
-Create the following OIDC connector [resource spec](../../reference/resources.mdx) as `gworkspace-connector.yaml`. We will explain how to choose values for fields within the resource spec below.
+Create the OIDC connector resource using `tctl`. We will explain how to choose
+values for fields within the resource spec below:
 
 <Tabs>
-  <TabItem scope={["oss", "enterprise"]} label="Self Hosted">
-    ```yaml
-    (!examples/resources/gworkspace-connector.yaml!)
-    ```
-  </TabItem>
-
-  <TabItem scope={["cloud"]} label="Teleport Cloud">
-    ```yaml
-    (!examples/resources/gworkspace-connector-inline.yaml!)
-    ```
-  </TabItem>
+<TabItem scope={["oss", "enterprise"]} label="Self Hosted">
+
+```code
+$ tctl sso configure oidc --preset google --id <CLIENT-ID> \
+--secret <CLIENT-SECRET> \
+--google-acc-uri <PATH/TO/SERVICE-ACCOUNT-KEY>.json \
+--claims-to-roles groups,auditor@example.com,auditor \
+--claims-to-roles groups,teleport-developers@example.com,access \
+--google-admin=<GOOGLE-WORKSPACE-ADMIN-EMAIL> > gworkspace-connector.yaml
+```
+
+The file created will resemble:
+
+```yaml
+kind: oidc
+metadata:
+  name: google
+spec:
+  claims_to_roles:
+  - claim: groups
+    roles:
+    - auditor
+    value: auditor@example.com
+  - claim: groups
+    roles:
+    - access
+    value: teleport-developers@example.com
+  client_id: CLIENT-ID
+  client_secret: CLIENT-SECRET
+  display: Google
+  google_admin_email: <GOOGLE-WORKSPACE-ADMIN-EMAIL>
+  google_service_account_uri: /PATH/TO/SERVICE-ACCOUNT-KEY.json
+  issuer_url: https://accounts.google.com
+  redirect_url: https://teleport.example.com/v1/webapi/oidc/callback
+version: v3
+```
+
+</TabItem>
+<TabItem scope={["cloud"]} label="Teleport Enterprise Cloud">
+
+```code
+$ tctl sso configure oidc --preset google --id CLIENT-ID \
+--secret CLIENT-SECRET \
+--claims-to-roles groups,auditor@example.com,auditor \
+--claims-to-roles groups,teleport-developers@example.com,access \
+--google-admin=<GOOGLE-WORKSPACE-ADMIN-EMAIL> \
+--google-acc '
+{
+  "type": "service_account",
+  ...
+  "universe_domain": "googleapis.com"
+}'
+```
+
+The file created will resemble:
+
+```yaml
+kind: oidc
+metadata:
+  name: google
+spec:
+  claims_to_roles:
+  - claim: groups
+    roles:
+    - auditor
+    value: auditor@example.com
+  - claim: groups
+    roles:
+    - access
+    value: teleport-developers@example.com
+  client_id: CLIENT-ID.apps.googleusercontent.com
+  client_secret: CLIENT-SECRET
```

Comment:
```suggestion
  client_id: <CLIENT-ID>.apps.googleusercontent.com
  client_secret: <CLIENT-SECRET>
```

Reply:
Actually, it should be: `client_id: <CLIENT-ID>`, since `.apps.googleusercontent.com` is included in what the user provides. Having it outside of the var here implies that `tctl` adds it.